src/app/onboard/page.tsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+
+import React from "react";
+import { useRouter } from "next/navigation";
+import { useUser } from "@auth0/nextjs-auth0/client";
+import { useForm, SubmitHandler } from "react-hook-form";
+import Badge from "../../components/Badge";
+import { UserIcon } from "lucide-react";
+
+type FormData = {
+  name: string;
+  organizationName: string;
+  organizationDomain: string;
+};
+
+const Onboard = () => {
+  const router = useRouter();
+  const { user, isLoading } = useUser();
+  const { register, handleSubmit } = useForm<FormData>();
+  const onSubmit: SubmitHandler<FormData> = async (data) => {
+    try {
+      const response = await fetch("/api/onboard", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(data),
+      });
+
+      if (response.ok) {
+        console.log("Successfully Done",response)
+      } else {
+        console.error("Failed to create tenant:", await response.text());
```

Comment:
Can we add the error somewhere in page so that the user knows what is the issue?

See guidance: https://nextjs.org/docs/app/building-your-application/routing/error-handling